application/app/src/main/java/com/example/liftvectr/ChartDisplay.java

```diff
@@ -0,0 +1,153 @@
+package com.example.liftvectr;
```

Comment:
Looks like the OG ChartDisplay on your branch wasn't removed after pulling in main and addressing merge conflicts.  Any chance you could add another commit removing it? (Don't want to cause confusion between this and the new /utils/ChartDisplay.